packages/block-editor-tools/package.json

```diff
@@ -14,18 +14,19 @@
     "npm": ">=8.0.0 <10.0.0"
```

Comment:
Can we remove the cap here? With how this project is configured I don't think we should be excluding newer versions of node and npm, especially now that node 20 is LTS